### PR TITLE
[l10n_xx] auto_apply on fiscal positions for all localizations

### PR DESCRIPTION
**Pull request by rgo-odoo**
Task: https://www.odoo.com/web?#id=1952975&action=333&active_id=131&model=project.task&view_type=form&menu_id=4720

### CODE REVIEW

File: addons/l10n_lu/i18n_extra/fr.po [hunk 1]
```diff
@@ -2057,6 +2057,11 @@ msgstr "Virements internes : solde créditeur"
 msgid "Internal transfers : debit balance"
 msgstr "Virements internes : solde débiteur"
 
+#. module: l10n_lu
+#: model:account.fiscal.position.template,name:l10n_lu.account_fiscal_position_template_private_LU_IC
+msgid "EU private"
+msgstr ""
```
Comment (jco-odoo) [inline addons/l10n_lu/i18n_extra/fr.po]: @rgo-odoo a bit weird to add translation but not the translation itself.  EU privé (fr)  EU privat (de)

File: addons/l10n_be/i18n_extra/nl.po [hunk 2]
```diff
@@ -1529,6 +1529,11 @@ msgstr "Waardeverminderingen op bestellingen in uitvoering"
 msgid "Réductions de valeur sur stocks"
 msgstr "Waardeverminderingen op voorraden"
 
+#. module: l10n_be
+#: model:account.fiscal.position.template,name:l10n_be.fiscal_position_template_5
+msgid "EU privé"
+msgstr ""
```
Comment (jco-odoo) [inline addons/l10n_be/i18n_extra/nl.po]: EU privé as translation

File: addons/l10n_ch/data/account_fiscal_position_data.xml [hunk 3]
```diff
@@ -2,10 +2,18 @@
 <odoo>
     <data noupdate="1">
         <!-- Fiscal Position Templates -->
+        <record id="fiscal_position_template_1" model="account.fiscal.position.template">
+            <field name="name">Schweiz National</field>
```
Comment (rgo-odoo) [inline addons/l10n_ch/data/account_fiscal_position_data.xml]: @jco-odoo Could you provide me German/Italian translation for this term ?

Comment (jco-odoo) [reply addons/l10n_ch/data/account_fiscal_position_data.xml]: It is in German.  English would be: Switzerland National and Italian Svizzera nazionale

**Comment by rgo-odoo** [comment]
@jco-odoo pushed my last changes. thanks.

**Comment by jco-odoo** [comment]
@rgo-odoo In the commit, there is only l10n_ca.  It should say l10n_* maybe

**Comment by jco-odoo** [comment]
@rgo-odoo Misread the commit.  Thanks!

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**